llvm/lib/Target/SPIRV/SPIRVCommandLine.cpp

```diff
@@ -68,7 +68,8 @@ static const std::map<std::string, SPIRV::Extension::Extension>
          SPIRV::Extension::Extension::SPV_KHR_shader_clock},
         {"SPV_KHR_cooperative_matrix",
          SPIRV::Extension::Extension::SPV_KHR_cooperative_matrix},
-};
+        {"SPV_KHR_non_semantic_info",
+         SPIRV::Extension::Extension::SPV_KHR_non_semantic_info}};
```

Comment:
Please update https://github.com/llvm/llvm-project/blob/main/llvm/docs/SPIRVUsage.rst accordingly.

Reply:
Done. Thank you for pointing this out.